base/uk.ac.stfc.isis.ibex.help/src/uk/ac/stfc/isis/ibex/help/Help.java

```diff
@@ -95,4 +97,9 @@ public void stop(BundleContext bundleContext) throws Exception {
 	public UpdatedValue<String> date() {
 		return date;
 	}
+
+    public String GetPVPrefix() {
```

Comment:
Java convention is to use camelcase but with a small first letter for method names. See https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/GUI-Coding-Conventions#general for a fuller explanation if needed.